src/applications/financial-status-report/components/InstallmentContract.jsx

```diff
@@ -0,0 +1,310 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { setData } from 'platform/forms-system/src/js/actions';
+import {
+  VaTextInput,
+  VaDate,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { parseISODate } from 'platform/forms-system/src/js/helpers';
+import { isValidCurrency } from '../utils/validations';
+
+const defaultRecord = [
+  {
+    contractType: '',
+    creditorName: '',
+    originalLoanAmount: '',
+    unpaidBalance: '',
+    amountDueMonthly: '',
+    from: '',
+    amountOverdue: '',
+  },
+];
```

Comment:
Could be an object since you are only using `defaultRecord[0], you can replace it with defaultRecord, but it works the way you have it as well so either way

```
const defaultRecord = {
    contractType: '',
    creditorName: '',
    originalLoanAmount: '',
    unpaidBalance: '',
    amountDueMonthly: '',
    from: '',
    amountOverdue: '',
};
```